pincer/objects/guild/guild.py

```diff
@@ -1538,6 +1502,91 @@ async def delete_template(
             construct_client_dict(self._client, data)
         )
 
+
+    async def list_stickers(self) -> AsyncIterator[Sticker]:
+        """|coro|
+        Yields sticker objects for the current guild.
+        Includes ``user`` fields if the bot has the ``MANAGE_EMOJIS_AND_STICKERS`` permission.
+
+        Yields
+        ------
+        :class:`~pincer.objects.message.sticker.Sticker`
+            a sticker for the current guild
+        """
+
+        for sticker in await self._http.get(f"guild/{self.id}/stickers"):
+            yield Sticker.from_dict(sticker)
+
+    async def get_sticker(self, _id: Snowflake) -> Sticker:
+        """|coro|
+        Returns a sticker object for the current guild and sticker IDs.
+        Includes the ``user`` field if the bot has the ``MANAGE_EMOJIS_AND_STICKERS`` permission.
+
+        Parameters
+        ----------
+        _id : int
+            id of the sticker
+
+        Returns
+        -------
+        :class:`~pincer.objects.message.sticker.Sticker`
+            the sticker requested
+        """
+        sticker = await self._http.get(f"guilds/{self.id}/stickers/{_id}")
+        return Sticker.from_dict(sticker)
+
+    async def create_sticker(
+        self,
+        name: str,
+        tags: str,
+        file,
+        description: str = "",
+        reason: Optional[str] = None,
+    ) -> Sticker:
+        """NOT IMPLEMENTED: DOES NOT WORK"""
+        raise NotImplementedError
+        # TODO: Fix this once File is fixed
+        # """|coro|
+        # Create a new sticker for the guild.
+        # Requires the ``MANAGE_EMOJIS_AND_STICKERS permission``.
+        #
+        # Parameters
+        # ----------
+        # name : str
+        #     name of the sticker (2-30 characters)
+        # tags : str
+        #     autocomplete/suggestion tags for the sticker (max 200 characters)
+        # file :
+        #     the sticker file to upload, must be a PNG, APNG, or Lottie JSON file, max 500 KB
+        # description : Optional[:class:`str`]
+        #     description of the sticker (empty or 2-100 characters) |default| :data:`""`
+        # reason : Optional[:class:`str`] |default| :data:`None`
+        #     reason for creating the sticker
+        #
+        # Returns
+        # -------
+        # :class:`~pincer.objects.message.sticker.Sticker`
+        #     the newly created sticker
+        # """
+        # sticker = await self._http.post(
+        #     f"guilds/{self.id}/stickers",
+        #     headers=remove_none({"X-Audit-Log-Reason": reason})
+        # )
+        #
+        # return Sticker.from_dict(sticker)
```

Comment:
This can be implemented now. I don't have time to explain rn, can say stuff tomorrow.

Reply:
@Lunarmagpie can you say stuff now?
If it's complicated, can you implement just this function on this branch?

Reply:
yeah, thanks for reminding me.

Reply:
It seems to work since I get the max stickers error. I don't have a guild to upload stickers to verify the error message is what it seems :skull: 
Turns out the discord devs decided to make stickers function differently than everything else so I had to manually use a multipart form from aiohttp.